File: shared-atlas-dart/lib/src/interfaces/create_session_dto.dart
/// Payload for creating a new chat session.
/// 创建新聊天会话的请求体。
/// 
/// API: `POST /api/chat/sessions`
class CreateSessionDto {

  CreateSessionDto({
  });

  factory CreateSessionDto.fromJson(Map<String, dynamic> json) {
    return CreateSessionDto(
    );
  }
  Map<String, dynamic> toJson() {
    return {
    };
  }
}

File: shared-atlas-rust/src/interfaces/create_session_dto.rs
use serde::{Serialize, Deserialize};

/// Payload for creating a new chat session.
/// 创建新聊天会话的请求体。
/// 
/// API: `POST /api/chat/sessions`
#[derive(Debug, Clone, Serialize, Deserialize)]
#[serde(rename_all = "camelCase")]
pub struct CreateSessionDto {
}